docs/web-player.md

```diff
@@ -134,6 +134,7 @@ See the general [Usage](#usage) section above.
 | `latency`       | R   | `number`                                                                                                                | `NaN`       | End to end latency in seconds.                                                     |
 | `playbackState` | R   | `'BUFFERING'` \| `'ENDED'` \| `'FAST_FORWARD'` \| `'PAUSED'` \| `'PLAYING'` \| `'REWIND'` \| `'SEEKING'` \| `'SLOW_MO'` | `'PAUSED'`  | Playback state.<br />Dispatches: `livery-playback-change`                          |
 | `quality`       | R   | `string`                                                                                                                | `null`      | Active quality label.<br />Dispatches: `livery-quality-change`                     |
+| `streamConfig`  | R   | `Config`                                                                                                                | `undefined` | Stream config.                                                                     |
```

Comment:
It may not be obvious to a customer-dev using the library what a 'stream config' is and what information they can retrieve from it. Please expand.

Reply:
As discussed, instead I'll nest the exposure of the private stream config property from the LiveryPlayer under the unsupported debug property and leave it out of the documentation here.